docs/docsite/rst/scenario_guides/guide_aws.rst

```diff
@@ -3,4 +3,8 @@
 Amazon Web Services Guide
 =========================
 
+.. important::
+
+  Red Hat Ansible Automation Platform will soon be available on Microsoft Azure. `Sign up to preview the experience <https://www.redhat.com/en/engage/ansible-microsoft-azure-e-202110220735>`_. 
+
```

Comment:
This feels a bit wrong to promote Azure from the AWS scenario guide.  I see nothing wrong with the additions to other pages.

```suggestion
```

Reply:
ok removed it, thanks!